src/runtime/config/configuration-clh.toml.in

```diff
@@ -39,8 +39,8 @@ rootfs_type=@DEFROOTFSTYPE@
 # Supported TEEs:
 # * Intel TDX
 #
-# Default false
-# confidential_guest = true
+# Default true
```

Comment:
Let's leave 'Default true'. The default value is still 'false' (if we were to comment or remove this line from the config, the default behavior would still be false (there's a different file/mechanism in which the defaults are defined)

Reply:
I apologize, I intended to say: "Let's leave 'Default false'" and not change the comment to 'true'